NEWS.md

```diff
@@ -1,33 +1,59 @@
+# sdtm.oak 0.1
+
+The V0.1 release of {sdtm.oak} users can create the majority of the SDTM domains. Domains that are *NOT* in scope for the V0.1 release are DM, Trial Design Domains, SV, SE, RELREC, Associated Person domains, and EPOCH Variable across all domains.
+
+-   Functions for commonly used SDTM mapping Algorithms
+    -   assign_no_ct\
+    -   assign_ct\
+    -   hardcode_no_ct\
+    -   hardcode_ct\
+    -   assign_datetime\
+    -   condition_add\
+-  Functions for SDTM derived variables\
+    -   Derive Baseline Flag or Last Observation Before Exposure Flag\
+    -   Derive the sequence number (--SEQ) variable\
+    -   Derive study day\
+    -   ISO8601 date, datetime conversion.\
+-  Functions to support {sdtm.oak}
+    - derive oak id variables
+    - Create conditioned dataframes to support if then else conditions in SDTM mappings
+    - Import study controlled terminology
+-  Articles
+    - Algorithms
+    - Creating an Events SDTM domain
+    - Creating a Findings SDTM domain
+    - Conditioned Data Frames
+    - Converting dates, times or date-times to ISO 8601
+    - Path to Automation
+
 # sdtm.oak 0.0.0.9005 (development version)
```

Comment:
I'm wonder if we should remove all these developer versions. In my opinion, everything should be on 0.1